string command
string arg1
string arg2
---
bool success
string message
